apps/provider-gateway/src/environments/environment.ts

```diff
@@ -1,15 +1,39 @@
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
-/* eslint-disable @typescript-eslint/naming-convention */
+
+const subdomain = process.env['NX_B2C_SUBDOMAIN'] || '';
+const clientId = process.env['NX_GATEWAY_C_ID'] || '';
+const gatewayApiUrl = 'https://api.provider.dev.dbhite.com';
+const portalApiUrl = 'https://api.portal.dev.dbhite.com';
+const authority = `https://${subdomain}.b2clogin.com/${subdomain}.onmicrosoft.com/b2c_1_sign_in_1`;
+const authorityDomain = `${subdomain}.b2clogin.com`;
+// const appInactiveTimeout = 900; // 15 minutes
+const appInactiveTimeout = 10; // 10 seconds
 
 export const environment = {
   production: false,
   UAT: false,
-  NX_GATEWAY_API: 'https://api.provider.dev.dbhite.com',
-  NX_PORTAL_API: 'https://api.portal.dev.dbhite.com',
-  B2C_SUBDOMAIN: process.env['NX_B2C_SUBDOMAIN'],
-  NX_GATEWAY_C_ID: process.env['NX_GATEWAY_C_ID'],
+  NX_GATEWAY_API: gatewayApiUrl,
+  NX_PORTAL_API: portalApiUrl,
+  B2C_SUBDOMAIN: subdomain,
+  NX_GATEWAY_C_ID: clientId,
+  clientId: clientId,
+  subdomain: subdomain,
+
+  msalConfig: {
+    auth: {
+      clientId: clientId,
+      authority: authority,
+      authorityDomain: authorityDomain,
+    },
+  },
+  // apiConfig: {
```

Comment:
uncomment apiConfig and set to suggestion:
```suggestion
  apiConfig: {
    scopes: ['provider.read'],
    uri: `https://${subdomain}.onmicrosoft.com/provider-api/`,
  },
```